app/0.14.x/configuration.md

```diff
@@ -1035,6 +1035,7 @@ record types. The `LAST` type means the
 type of the last successful lookup (for the
 specified name). The format is a (case
 insensitive) comma separated list.
+Remove any record types that Kong should ignore. 
```

Comment:
Consider mentioning that once Kong reverts to the `LAST` record type for a particular service, it no longer performs DNS queries for the service regardless of the TTL.

Reply:
I don't think that comment is accurate. Anyway this is the short description, so let's keep it short.

Reply:
@Tieske can you clarify what the effect is of removing a record type from this list? I see you removed the sentence I added here.